spec/models/user_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it { should validate_presence_of(:name) }
+  it { should validate_numericality_of(:postCounter).only_integer.is_greater_than_or_equal_to(0) }
+end
```

Comment:
- [ ] Good job on validations and adding test spec for `user` and `post` models. However, let's also endeavor to add test specs for `comment` and `like` models.

Reply:
Unit specs added for comment and like models. Thanks :)